bubble_sort.rb

```diff
@@ -0,0 +1,49 @@
+# Internal: Swaps indexed element of an array with the next element.
+# Doesn't check if the elements exist. Modifies the array.
+#
+# %Array - Array to modify
+# %i - index of the element to be swapped
```

Comment:
Remove all these commented codes. Always Think like you are pushing the project into production.

Reply:
Hello, thanks for the review.
I'm afraid I have to disagree with this part of the review. This snippet doesn't show commented code, but documentation following the [TomDoc](http://tomdoc.org/) format. We were following the ruby code [style guide](https://github.com/github/rubocop-github/blob/master/STYLEGUIDE.md#ruby-style-guide) linked from the project specification.

Reply:
I really appreciate that you used the TomDoc documentation format. But I suggest using this in the development phase. It is primarily meant for use during the development of tests.